### PR TITLE
(chore) set ui-builder as a standalone

### PR DESCRIPTION
**Pull request by hebus**
* set all components/directives/pipes as standalone
* update demo app accordingly

### CODE REVIEW

File: projects/lib/src/configurator/editors/condition-editor.component.html [hunk 1]
```diff
@@ -8,29 +8,29 @@
 
   <div class="mb-2">
     <label class="form-label" for="data" i18n>Data</label>
-    <select [(ngModel)]="context.config.condition.data" id="data" class="form-select" (ngModelChange)="updateConfig()">
+    <select [(ngModel)]="context.config?.condition.data" id="data" class="form-select" (ngModelChange)="updateConfig()">
```
Comment (ericleib) [inline projects/lib/src/configurator/editors/condition-editor.component.html:11]: There is a `*ngIf="context.config.condition"` above which means that if `context.config` is undefined there will be an error anyway.

File: projects/lib/src/configurator/toolbar/toolbar.component.html [hunk 2]
```diff
@@ -1,14 +1,14 @@
-<div class="btn-group uib-toolbar uib-toolbar-anim" *ngIf="{enabled: configurableService.editorEnabled$ | async} as editor" >
+<div class="btn-group uib-toolbar uib-toolbar-anim text-white" *ngIf="{enabled: configurableService.editorEnabled$ | async} as editor" >
   <ng-container *ngIf="editor.enabled">
     <ng-container [ngTemplateOutlet]="template"></ng-container>
-    <button class="btn btn-primary" (click)="configService.undo()" [ngClass]="{'disabled': !(configService.canUndo$()| async)}" uib-tooltip="Undo" i18n-uib-tooltip placement="top">
+    <button class="btn btn-primary text-white" (click)="configService.undo()" [ngClass]="{'disabled': !(configService.canUndo$()| async)}" uib-tooltip="Undo" i18n-uib-tooltip placement="top">
       <svg-icon key="undo"></svg-icon>
     </button>
-    <button class="btn btn-primary" (click)="configService.redo()" [ngClass]="{'disabled': !(configService.canRedo$()| async)}" uib-tooltip="Redo" i18n-uib-tooltip placement="top">
+    <button class="btn btn-primary text-white" (click)="configService.redo()" [ngClass]="{'disabled': !(configService.canRedo$()| async)}" uib-tooltip="Redo" i18n-uib-tooltip placement="top">
```
Comment (ericleib) [inline projects/lib/src/configurator/toolbar/toolbar.component.html:7]: Same comment as above regarding `btn-primary`

File: projects/lib/src/dynamic-views/dynamic-views.module.ts [hunk 3]
```diff
@@ -1,16 +1,11 @@
...
-import { UtilsModule } from '../utils/utils.module';
 import { ConfigModule } from '../configuration';
-import { ZoneComponent } from './zone/zone.component';
-import { ItemComponent } from './item/item.component';
 
 @NgModule({
-  imports: [CommonModule, DndModule, ConfigurableModule, UtilsModule, ConfigModule],
-  declarations: [ZoneComponent, ItemComponent],
-  exports: [UtilsModule, ZoneComponent],
+  imports: [DndModule, ConfigModule],
```
Comment (ericleib) [inline projects/lib/src/dynamic-views/dynamic-views.module.ts:7]: Do you still need this module ? It does not declare or export anything, so I don't think it's needed.

File: projects/lib/src/utils/svg-icon/svg-icon.component.ts [hunk 4]
```diff
@@ -64,9 +65,10 @@ export class SvgIconComponent {
   private mergedConfig: SVG_CONFIG;
   private lastKey!: string;
 
+  private registry: SvgIconRegistry = inject(SvgIconRegistry);
```
Comment (ericleib) [inline projects/lib/src/utils/svg-icon/svg-icon.component.ts:68]: Is there a reason for using `inject()` here?

File: projects/lib/src/dynamic-views/dynamic-views.module.ts (hunk 3 above)
Comment (hebus) [reply projects/lib/src/dynamic-views/dynamic-views.module.ts:7]: you right, no need anymore

**Review by ericleib — APPROVED** [review]

**Pull request merged**